complex_envs/active_damping_env.py

```diff
@@ -31,21 +31,24 @@ def __init__(self,config):
             config:  A dict containing parameters for the system
```

Comment:
doc string insufficiently descriptive

Reply:
Good catch, I will add one this evening.